docs/src/intro-js.md

```diff
@@ -81,30 +82,122 @@ The `tests` folder contains a basic example test to help you get started with te
 
 By default tests will be run on all 3 browsers, chromium, firefox and webkit using 3 workers. This can be configured in the [playwright.config file](./test-configuration.md). Tests are run in headless mode meaning no browser will open up when running the tests. Results of the tests and test logs will be shown in the terminal.
 
+<Tabs
+  defaultValue="npm"
+  values={[
+    {label: 'npm', value: 'npm'},
+    {label: 'yarn', value: 'yarn'},
+    {label: 'pnpm', value: 'pnpm'}
+  ]
+}>
+<TabItem value="npm">
+
 ```bash
 npx playwright test
 ```
 
+</TabItem>
+
+<TabItem value="yarn">
+
+```bash
+yarn run playwright test
```

Comment:
Lets align with create-playwright?

yarn playwright and pnpm exec playwright.

https://github.com/microsoft/create-playwright/blob/d278d04bb8d2e45a88e973f39ce6a2b3126da7c3/src/packageManager.ts#L65

Reply:
agreed pushed the fix.